spread.yaml

```diff
@@ -9,7 +9,6 @@ environment:
     PATH: $GOPATH/bin:/snap/bin:$PATH:/usr/lib/go-1.6/bin
     TESTSLIB: $PROJECT_PATH/tests/lib
     SNAPPY_TESTING: 1
-    SNAP_REEXEC: 0
```

Comment:
after further thinking, I think is very strange to run our main tests for master using reexec on... we really need a 2nd suite I think or a spread-cron task I think.

I mean our main suite should run SNAP_REEXEC=0 because it's about testing the code we are writing atm